packages/cadl-ranch-specs/http/azure/core/basic/main.tsp

```diff
@@ -70,6 +70,20 @@ model UserExportParams {
   format: string;
 }
 
+model SwapUsersOptions {
+  @doc("Represents the first user id.")
+  firstUserId: int32;
+
+  @doc("Represents the second user id.")
+  secondUserId: int32;
+}
+
+model SwapUsersParams {
```

Comment:
How is this test related to "swap user"?

Reply:
I have made improvements to this method, instead of simply calling it for the user. Regarding the swapParameter, I think it serves as a credential indicating which two models to swap, and it cannot be modified for the time being.